packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -263,7 +263,7 @@ function SendConfirmation(props: Props) {
         confirmButton={{
           action: onSendClick,
           text: isInvite ? t('sendAndInvite') : t('send'),
-          disabled: isSending || !!feeError,
```

Comment:
Should we really move forward if we couldn't calculate the fee?

Reply:
Well, I changed this thinking of my own personal case, I would prefer to be able to send the tx even if fee estimation fails, but I guess that might not be true to everyone and we shouldn't change it without discussing, I'll revert this